lib/query/limit.ex

```diff
@@ -0,0 +1,25 @@
+defmodule QueryBuilder.Query.Limit do
+  @moduledoc false
+
+  require Ecto.Query
+
+  def limit(ecto_query, _assoc_fields, value) do
+    apply_limit(ecto_query, value)
+  end
+
+  defp apply_limit(query, value) when is_integer(value) do
+    Ecto.Query.limit(query, ^value)
+  end
+
+  defp apply_limit(query, value) do
+    case Integer.parse(value) do
+      {int_value, __rest} ->
```

Comment:
why the double underscore here? I think one underscore is more idiomatic in elixir for unused variables (just a small detail:))

Reply:
Yup.  Typo!